pkg/agent/config.go

```diff
@@ -221,6 +220,10 @@ type Config struct {
 	// FilterTCPFlags is the TCP flags to filter flows.
 	// possible values are: SYN, SYN-ACK, ACK, FIN, RST, PSH, URG, ECE, CWR, FIN-ACK, RST-ACK
 	FilterTCPFlags string `env:"FILTER_TCP_FLAGS"`
+	// EnableNetworkEventsMonitoring enables monitoring network plugin events, default is false.
+	EnableNetworkEventsMonitoring bool `env:"ENABLE_NETWORK_EVENTS_MONITORING" envDefault:"false"`
+	// NetworkEventsMonitoringGroupID to allow ebpf hook to process samples for specific groupID and ignore the rest
+	NetworkEventsMonitoringGroupID int `env:"NETWORK_EVENTS_MONITORING_GROUP_ID" envDefault:"10"`
```

Comment:
what's exactly this group ID, how are we supposed to know what to set?

Reply:
so this is somehow tied to an OVN configuration that is done somewhere else?

Reply:
yeah in ovn and currently hardcoded to `10`

Reply:
here is the slack thread about it for reference
```sh
amorenoz
@Nadia Pinaeva @mmahmoud
 I'd keep it simple for the time being, how about this:
ovnk-observ-library:
The API allows the user to provide a group and an external_id, e.g: configureCollector(id, 10, "ebpfAgent")
The API allows the user to clean up the configuration when it is closed normally.
The library checks if the entry already exists:
If it exists, it may mean that ebpfAgent crashed and did not clean up resources, the library checks if the external id is the same.
If it is, then the configuration is kept and returns OK (or maybe a code indicating that it already existed but it's not an error)
If it's not, return an error, someone stole the group from NetObserv, it's a bug [1]
If not, the library configures and returns OK.
ebpfAgent
Hardcodes and documents the group id that it'll use
Configures it via library
ebpf code just checks against that number.
[1] We could be smarter and nave ebpfAgent generate another number and retry here but that'd mean  ebpfAgent has to deal with dynamic ids which is a bit more complicated.
Apart from error cases like [1], in the future we may want  to configure more than one collector, i.e: more than one group_id, for NetObserv. So at some point we may need to support injecting them dynamically into the ebpf agent through a MAP or something. But I think right now is not strictly needed and we could save some devel time here.
```